readthedocs/invitations/templates/invitations/invitation_detail.html

```diff
@@ -0,0 +1,70 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block title %}
+  {% trans "Accept or decline invitation" %}
+{% endblock %}
+
+{% block content %}
+<h2>
+  {% url "profiles_profile_detail" invitation.from_user.username as profile_url %}
+  {% with from_name=invitation.from_user.get_full_name|default:invitation.from_user.username %}
+    {% if invitation.object_type == "project" %}
+      {% url "projects_detail" invitation.object.slug as project_url %}
+
+      {% blocktrans trimmed with from_name=from_name profile_url=profile_url project_url=project_url project_name=invitation.object.name %}
```

Comment:
I have implemented your suggestion, but still needed to check if the object was a team, because when it is a team I'm displaying `has inveted you to the {team} team from the {organization} organization.`, if  you have a suggestion of another way to present a team, let me know. Maybe `has invited you to the {organization / {team} team`?

Reply:
Seems you could note the organization in the target name itself, instead of breaking up the sentence:

```
target_name = "Read the Docs Read-only team"
```

Reply:
Also, thinking about this more, is there a reason we want links in this at all? If the user doesn't yet have access to the organization, all these links will 404 for that user. It's only in the case that the user is in the organization already that the links will work.

This might be an area instead rely on UI elements for the requesting user, organization, and team, instead of using text links. We can come back to this one with new templates perhaps, the text works for now.

Reply:
Mostly so users can distinguish the name from normal text, and also the URL has the slug, so they know the organization is the real one. Maybe just show the slug in code tags?

Reply:
Well, I guess thinking medium to long term, the overlap here is going to be using dedicated UI elements for each of these -- requesting user, the organization, and a team if applicable. So, instead of one sentence that combines all of these, these are best off as independent blocks for each element.

I don't have anything in mind yet for the new interface, but would aim for something like GitHub's interface for invite acceptance:

![image](https://user-images.githubusercontent.com/1140183/181655043-478e77f9-a99b-4f55-b795-6131f5fa860b.png)

That is, organization, team, and maybe user would all be rich elements, with team/org metadata/images/etc. And I'd keep sentences separate to avoid complex logic.

Reply:
I have updated the text to use plain slugs, and removed the linking (linking only makes sense for invitations to projects on .org).